play_tik_tak_toe.py

```diff
@@ -15,6 +15,28 @@ def SwitchTurns():
         print("O")
         isXTurn = True
 
+def ConvertGridToIndex(gridSpace):
+    if gridSpace == 1:
+        indexElement = 1
+    if gridSpace == 2:
+        indexElement = 5
+    if gridSpace == 3:
+        indexElement = 9
+    if gridSpace == 4:
+        indexElement = 13
+    if gridSpace == 5:
+        indexElement = 17
+    if gridSpace == 6:
+        indexElement = 21
+    if gridSpace == 7:
+        indexElement = 25
+    if gridSpace == 8:
+        indexElement = 29
+    if gridSpace == 9:
+        indexElement = 33
+    return indexElement
+   
+#print(grid[33])
```

Comment:
why did you leave this comment in?

Reply:
thats a good question. I have it as a double check for the index output aligning on the right element. if any future changes are needed it may be useful as a quick self check but is not necessary for the code

Reply:
ahh that's a good reason!